aggregator/src/aggregator.rs

```diff
@@ -2173,6 +2186,39 @@ impl VdafOps {
                         if aggregation_job.last_request_hash()
                             != existing_aggregation_job.last_request_hash()
                         {
+                            if log_forbidden_mutations {
+                                let span = debug_span!(
+                                    parent: Span::current(),
+                                    "aggregation_job_init_forbidden_mutation_span",
+                                );
+                                let _span_entered = span.enter();
+                                let existing_report_metadatas: Vec<_> = tx
+                                    .get_report_aggregations_for_aggregation_job(
+                                        vdaf.as_ref(),
+                                        &Role::Helper,
+                                        task.id(),
+                                        aggregation_job.id(),
+                                    )
+                                    .await?
+                                    .iter()
+                                    .map(|ra| ra.report_metadata())
+                                    .collect();
+                                let incoming_request_report_metadatas: Vec<_> = report_share_data
+                                    .iter()
+                                    .map(|rsd| rsd.report_share.metadata())
+                                    .collect();
+                                tracing::event!(
+                                    Level::DEBUG,
```

Comment:
If this is a config option, we might as well use an `INFO` log IMO.

Reply:
Yes, I suppose that's simpler.